lib/spack/spack/package.py

```diff
@@ -1669,8 +1685,8 @@ def build_process():
                 # Ensure the metadata path exists as well
                 mkdirp(spack.store.layout.metadata_path(self.spec), mode=perms)
 
-            # Fork a child to do the actual installation
-            # we preserve verbosity settings across installs.
+            # Fork a child to do the actual code installation.
```

Comment:
I think the "code" part is leftover from when no-src packages couldn't have an `install` method.